lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -538,15 +554,17 @@ Expr RawMemManager::loadValueFromMem(PtrTy ptr, MemValTy mem,
   return res;
 }
 
-Expr RawMemManager::storeValueToMem(Expr _val, PtrTy ptr, MemValTy mem,
-                                    const llvm::Type &ty, uint32_t align) {
-  assert(ptr);
+RawMemManager::MemValTy RawMemManager::storeValueToMem(Expr _val, PtrTy ptr,
+                                                       MemValTy mem,
+                                                       const llvm::Type &ty,
+                                                       uint32_t align) {
+  assert(ptr.toExpr());
   Expr val = _val;
   const unsigned byteSz =
       m_sem.getTD().getTypeStoreSize(const_cast<llvm::Type *>(&ty));
-  ExprFactory &efac = ptr->efac();
+  ExprFactory &efac = ptr.toExpr()->efac();
 
-  Expr res;
+  MemValTy res = MemValTy(m_ctx.alu().getFalse());
```

Comment:
different from previous behavior. Used to return `Expr(0)` when nothing was stored.

Reply:
Fixed.